### PR TITLE
Jira DOC-703: Update Database backup to reflect start time and updated directions.

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

File: content/rs/administering/import-export/database-backup.md [hunk 1]
```diff
@@ -6,23 +6,26 @@ alwaysopen: false
...
 ---
-You can manually [export your data]({{< relref "/rs/administering/import-export/exporting-data.md" >}})
-from a specific Redis Enterprise Software database at any time.
-You can also schedule backups of your databases to make sure you always have valid backups.
-The backup process can be scheduled for every 1, 4, 12 or 24 hours from the time that you save the backup configuration.
 
-You can schedule backups to these locations:
+Periodic backups provide a way to restore data with minimal data loss.  With Redis Enterprise Software, you can schedule periodic backups to occur every once a day (every 24 hours), twice a day (every twelve hours), every four hours, or every hour.
+
+As of v6.2.8, you can specify the start time for twenty four or twelve hour backups.
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:12]:
```suggestion
Periodic backups provide a way to restore data with minimal data loss.  With Redis Enterprise Software, you can schedule periodic backups to occur once a day (every 24 hours), twice a day (every 12 hours), every four hours, or every hour.

As of v6.2.8, you can specify the start time for 24- or 12-hour backups.
```

File: content/rs/administering/import-export/database-backup.md [hunk 2]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+    {{<image filename="images/rs/database-configuration-periodic-backup.png" alt="In the admin console, the Periodic backup settings can be found on the Configuration details tab of the database." >}}{{< /image >}}
+    
+6.  Use the following table to help specify the details:
+
+    | Control name | Description |
+    |--------------|-------------|
+    | **Interval** | Specifies the frequency of the backup; that is, the time between each backup snapshot.<br/><br/>Supported values include _Every 24 hours_, _Every 12 hours_, _Every 4 hours_, and _Every hour_. |
+    | **Set starting time** | _v6.2.8 or later:&nbsp;_ Specifies the start time for the backup; available when **Interval** is set to _Every 24 hours_ or _Every 12 hours_.<br/><br/>If not specified, defaults to a time selected by Redis Enterprise Software. |
+    | **Choose storage type** | Specifies the storage type for the backup.  Supported options vary and might require additional details.  To learn more, see [Supported storage types](#supported-storage-types).
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:63]: Mismatched anchor
```suggestion
    | **Choose storage type** | Specifies the storage type for the backup.  Supported options vary and might require additional details.  To learn more, see [Supported storage locations](#supported-storage-locations).
```

File: content/rs/administering/import-export/database-backup.md [hunk 3]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+7.  Select **Update** to apply your changes.
+
+Access to the storage location is verified when you apply your updates.  This means the location, credentials, and other details must exist and function before you can enable periodic backups.
+
+## Default backup start time
+
+If you do _not_ specify a start time for twenty four or twelve hour backups, Redis Enterprise Software chooses one for you, based on the time the backups are enabled.
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:71]:
```suggestion
If you do _not_ specify a start time for 24- or 12-hour backups, Redis Enterprise Software chooses one for you, based on the time the backups are enabled.
```

File: content/rs/administering/import-export/database-backup.md [hunk 4]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+This choice assumes that your database is deployed to a multi-tenant cluster containing multiple databases.  This means that default start times are staggered (offset) to ensure availability.  This is done by calculating a random offset which specifies a number of seconds added to the start time.  
+
+Here's how it works:
+
+- Assume you're enabling the backup at 4:00 pm (1600 hours).
+- You choose to back up your database every 12 hours.
+- Because you didn't set a start time, the cluster randomly chooses an offset of 4,320 seconds (or 72 minutes).
+
+This means your first periodic backup will run roughly around 5:12 pm (72 minutes after the time you enabled backups) and repeats every twelve hours at about that same time.
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:81]:
```suggestion
This means your first periodic backup will run roughly around 5:12 pm (72 minutes after the time you enabled backups) and repeats every 12 hours at about that same time.
```

File: content/rs/administering/import-export/database-backup.md [hunk 5]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+- The SSH private keys are specified correctly.  You can use the key generated by the cluster or specify a custom key.
+
+    When using the cluster auto generated key, copy the **Cluster SSH Public Key** to the appropriate location on the SFTP server.  This is available from the **General** tab of the **Settings** menu in the admin console.
+
+    Use the server documentation to determine the appropriate location for the SSH Public Key.
+
+To backup to an SFTP server, enter the SFTP server location in the format:
+
+```sh
+sftp://user:password@host:<:custom_port>/path/
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:141]:
```suggestion
sftp://user:password@host:<custom_port>/path/
```

File: content/rs/administering/import-export/database-backup.md [hunk 6]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+```
+
+For example: `sftp://username:password@10.1.1.1/home/backups/`
+
+### Local mount point
+
+Before you choose to backup to a local mount point, make sure that:
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:148]:
```suggestion
Before you choose to back up to a local mount point, make sure that:
```

File: content/rs/administering/import-export/database-backup.md [hunk 7]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+- The node has network connectivity to the destination server of the mount point.
+- The `redislabs:redislabs` user has read and write privileges on the local mount point
+and on the destination server.
+- The backup location has enough disk space for your backup files. The backup files
+are saved with filenames that include the timestamp so that backup files are not overwritten.
+
+To backup to a local mount point for a node:
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:156]:
```suggestion
To back up to a local mount point for a node:
```

File: content/rs/administering/import-export/database-backup.md [hunk 8]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+    1.  Set **AWS Region** to an appropriate region.
+
+    1.  Set other properties according to your company standards.
+
+    1.  When finished, select the **Create bucket** button near the bottom of the screen.
+
+1.  Use the Buckets list to locate and select your bucket.  When the settings appear, select the **Permissions** tab, locate the **Access control list (ACL)** section, and then select the **Edit** button.
+
+1.  When the **Edit access control list (ACL)** screen appears, locate the Access for other AWS accounts section and then select the **Add grantee** button.
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:194]:
```suggestion
1.  When the **Edit access control list (ACL)** screen appears, locate the **Access for other AWS accounts** section and then select the **Add grantee** button.
```

File: content/rs/administering/import-export/database-backup.md [hunk 9]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+    1.  In the **Grantee** field, enter:
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:196]:
```suggestion
    1.  In the **Grantee** field, enter the AWS account ID:
```

File: content/rs/administering/import-export/database-backup.md [hunk 10]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+    
+    ```
+    fd1b05415aa5ea3a310265ddb13b156c7c76260dbc87e037a8fc290c3c86b614
+    ```
+
+    1.  In the **Objects** list, enable **Write**.
+    1.  In the **Bucket ACL** list, enable **Read** and **Write**.
+    1.  When finished, select the **Save changes** button.
+
+Once the bucket is available and the permissions are set, use the name of your bucket as the **Backup destination** for your database's Remote backup settings. For example, suppose your bucket is named *backups-bucket*.  In that case, set **Backup destination** to `s3://backups-bucket`.
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:206]:
```suggestion
Once the bucket is available and the permissions are set, use the name of your bucket as the **Backup destination** for your database's **Remote backup settings**. For example, suppose your bucket is named *backups-bucket*.  In that case, set **Backup destination** to `s3://backups-bucket`.
```

File: content/rs/administering/import-export/database-backup.md [hunk 11]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+### GCP Storage 
+
+For [Google Cloud Platform (GCP)
+console](https://developers.google.com/console/) subscriptions, store your backups in a Google Cloud Storage bucket:
+
+1. Sign in to Google Cloud Platform console.
+
+1. In the admin console menu, locate the _Storage_ section than select **Cloud Storage&nbsp;>&nbsp;Browser**.
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:215]:
```suggestion
1. In the admin console menu, locate the _Storage_ section then select **Cloud Storage&nbsp;>&nbsp;Browser**.
```

File: content/rs/administering/import-export/database-backup.md [hunk 12]
```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
...
+
+1. [Create a container](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#create-a-container) if you do not already have one
+
+1. [Manage storage account access keys](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage)
+
+Set your resource's **Backup Path** to the path of your storage account.
+
+The syntax for creating the backup varies according to your authorization mechanism.  For example:
+
+`abs://:storage_account_access_key@storage_account_name/container_name/[path/]`
```
Comment (rrelledge) [inline content/rs/administering/import-export/database-backup.md:257]:
```suggestion
`abs://storage_account_access_key@storage_account_name/container_name/[path/]`
```

**Review by rrelledge — APPROVED** [review]
Looks good. I left some suggestions for possible typos.

File: content/rs/administering/import-export/database-backup.md (hunk 12 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:257]: Fixed.

File: content/rs/administering/import-export/database-backup.md (hunk 11 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:215]: Fixed.

File: content/rs/administering/import-export/database-backup.md (hunk 10 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:206]: Reworked.

File: content/rs/administering/import-export/database-backup.md (hunk 9 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:196]: Fixed.

File: content/rs/administering/import-export/database-backup.md (hunk 8 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:194]: Fixed.

File: content/rs/administering/import-export/database-backup.md (hunk 7 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:156]: Fixed (and cleaned up a bit of nearby language).

File: content/rs/administering/import-export/database-backup.md (hunk 6 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:148]: Fixed.

File: content/rs/administering/import-export/database-backup.md (hunk 5 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:141]: Fixed-ish.  (There should only be one colon).

File: content/rs/administering/import-export/database-backup.md (hunk 4 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:81]: Reworked.

File: content/rs/administering/import-export/database-backup.md (hunk 3 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:71]: Fixed to intention

File: content/rs/administering/import-export/database-backup.md (hunk 2 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:63]: Fixed

File: content/rs/administering/import-export/database-backup.md (hunk 1 above)
Comment (lanceleonard) [reply content/rs/administering/import-export/database-backup.md:12]: Fixed to match intenntion.

**Review by kaitlynmichael — APPROVED** [review]

**Pull request merged**